x-pack/plugins/security_solution/public/explore/network/components/network_top_countries_table/columns.tsx

```diff
@@ -59,31 +54,20 @@ export const getNetworkTopCountriesColumns = (
       const id = escapeDataProviderId(`${tableId}-table-${flowTarget}-country-${geo}`);
       if (geo != null) {
         return (
-          <DraggableWrapper
+          <CellActions
             key={id}
-            dataProvider={{
-              and: [],
-              enabled: true,
-              id,
-              name: geo,
-              excluded: false,
-              kqlQuery: '',
-              queryMatch: { field: geoAttr, value: geo, operator: IS_OPERATOR },
+            mode={CellActionsMode.HOVER}
+            visibleCellActions={5}
+            showActionTooltips
+            triggerId={CELL_ACTIONS_DEFAULT_TRIGGER}
+            field={{
+              name: geoAttr,
+              value: geo,
+              type: 'keyword',
             }}
-            isAggregatable={true}
-            fieldType={'keyword'}
-            render={(dataProvider, _, snapshot) =>
-              snapshot.isDragging ? (
-                <DragEffects>
-                  <Provider dataProvider={dataProvider} />
-                </DragEffects>
-              ) : (
-                <>
-                  <CountryFlagAndName countryCode={geo} />
```

Comment:
Looks like we lost the flag, is there a reason we cannot support the flag anymore?

Reply:
https://github.com/elastic/kibana/pull/149460